CHANGELOG.md

```diff
@@ -1,7 +1,49 @@
 # Change Log
 All notable changes to the "vscode-azurearmtools" extension will be documented in this file.
 
-## Version 0.8.0 (2019-10-01)
+## Coming Soon
+- Support for variable iteration ("COPY blocks")
+
+## Version 0.8.0 (2019-10-28)
+### Added
+- Greatly improved schema errors and completion
+  - Our new language server now has a better understanding of Azure Resource Manager templates and can therefore provide a better error and completion experience beyond that provided using standard JSON validation
+  - It now narrows down its schema validation based on the resource type and apiVersion properties of each resource.
+  - No longer suggests changing the resource type if the apiVersion is not supported, or if a property is missing or invalid
```

Comment:
I think we can remove this. My change to remove missing 'apiVersion' if apiProfile and type are specified (and if we are able to get apiVersion from cache) is not in yet.

@jimmylewis , do you agree?

Reply:
I think this is from the ambiguous resolution issue we used to have.  If two schemas matched the other properties equally poorly, we'd flag apiVersion or resourceType as incorrect (even though they were valid).  Now with the new evaluation, we validate those first, then only flag any other properties that remain incorrect with respect to these two.

I think this is different from the apiProfile fix that you have in progress.

Reply:
Thanks for the clarification Jimmy!